bin/release

```diff
@@ -36,8 +36,10 @@ node bin/create-cli-duplicate-package.js
 # Publish the packages
 pnpm changeset publish --tag $tag
 
+# if [ "$tag" = "latest" ]; then
```

Comment:
this `if` is disabled until we can verify that the scripts work for nightly/experimental, then they should only be run for `latest`